sct-commons/src/main/java/org/lfenergy/compas/sct/commons/dto/ResumedDataTemplate.java

```diff
@@ -340,4 +342,12 @@ public ResumedDataTemplate setVal(String daiValue) {
         this.setDaiValues(List.of(newDaiVal));
         return this;
     }
+
+    public Optional<String> findFirstValue() {
+        Collection<String> values = this.getDaName().getDaiValues().values();
```

Comment:
use JAVA 8 : 
```java
return this.getDaName().getDaiValues().values()
                .stream()
                .findFirst();
```

Reply:
OK. This method has been fixed based on a comment of @AliouDIAITE. And also, now it uses Java 8.